net-gateway/src/authorization/mod.rs

```diff
@@ -16,100 +23,7 @@ pub async fn authorize(req: HttpRequest, verifier: impl net_token_verifier::veri
 		return Err(HttpResponse::Unauthorized().body("Unauthorized: Bearer token expected"));
 	};
 	match verifier.verify_token(token).await {
-		Ok(_) => Ok(token.to_string()),
+		Ok(token) => Ok(token),
 		Err(message) => Err(HttpResponse::Unauthorized().body(format!("Unauthorized: {}", message))),
 	}
-}
-
-#[cfg(test)]
```

Comment:
I guess you could just update these tests :(

Reply:
Hah, I mean, yeah
Will add such task into a backlog